api/src/main/java/org/apache/iceberg/actions/DeleteOrphanFiles.java

```diff
@@ -80,6 +81,41 @@ public interface DeleteOrphanFiles extends Action<DeleteOrphanFiles, DeleteOrpha
    */
   DeleteOrphanFiles executeDeleteWith(ExecutorService executorService);
 
+  /**
```

Comment:
Javadoc don't respect new lines. That's why explicit `<p>` are required. Also, it is a very fragile place, so I'd add as much documentation as possible. What about this?

```
  /**
   * Passes a prefix mismatch mode that determines how this action should handle situations when
   * the metadata references files that match listed/provided files except for authority/scheme.
   * <p>
   * Possible values are "ERROR", "IGNORE", "DELETE". The default mismatch mode is "ERROR",
   * which means an exception is thrown whenever there is a mismatch in authority/scheme.
   * It's the recommended mismatch mode and should be changed only in some rare circumstances.
   * If there is a mismatch, use {@link #equalSchemes(Map)} and {@link #equalAuthorities(Map)}
   * to resolve conflicts by providing equivalent schemes and authorities. If it is impossible
   * to determine whether the conflicting authorities/schemes are equal, set the prefix mismatch
   * mode to "IGNORE" to skip files with mismatches. If you have manually inspected all conflicting
   * authorities/schemes, provided equivalent schemes/authorities and are absolutely confident
   * the remaining ones are different, set the prefix mismatch mode to "DELETE" to consider files
   * with mismatches as orphan. It will be impossible to recover files after deletion,
   * so the "DELETE" prefix mismatch mode must be used with extreme caution.
   *
   * @param mode mode for handling prefix mismatches
   * @return this for method chaining
   */
```

Reply:
done